File: DPBO-teori/1 Pendahuluan Pemrograman Berorientasi Objek, Mampu memahami dan mengimplementasikan kelas dan merelasikannya/Balok java/Tabel.java
public class Tabel {
    private int baris;
    private int kolom;

    Tabel(){

    }
    Tabel(int baris, int kolom){
        this.baris=baris;
        this.kolom=kolom;
    }
    public void set_baris(int baris){
        this.baris=baris;
    }
    public void set_kolom(int kolom){
        this.kolom=kolom;
    }
    public int get_baris(){
        return this.baris;
    }
    public int get_kolom(){
        return this.kolom;
    }
    public void buat_baris(String[] isi, int add){
        int i=0,j=0;
        for(i=0; i<isi.length; i++){
            for(j=0; j<isi[i].length()+add; j++){
                System.out.print("-");
            }
        }
        System.out.println((""));
        for(i=0; i<isi.length; i++){
            System.out.print("| ");
            System.out.print(isi[i]);
            for(j=0;j<add-3;j++){
                System.out.print(" ");
            }
            System.out.print("|");
        }
        System.out.println("");
        for(i=0; i<isi.length; i++){
            for(j=0; j<isi[i].length()+add; j++){
                System.out.print("-");
            }
        }
        System.out.println((""));
    }
}

File: DPBO-praktikum/1/Python/Tabel.py
class Tabel:
    def __init__(self, baris=0, kolom=0):
        self.baris = baris
        self.kolom = kolom

    def set_baris(self, baris):
        self.baris = baris

    def set_kolom(self, kolom):
        self.kolom = kolom

    def get_baris(self):
        return self.baris

    def get_kolom(self):
        return self.kolom

    def buat_baris(self, isi, max, garis, type):#membuat tampilan baris tabel
        for i in range(garis + 13):#buat garis atas
            print(type, end="")
        print("")
        for i in range(len(isi)):
            print("| ", end="")
            print(isi[i], end=" ")#tampilkan isi data
            for j in range(max[i] - len(isi[i])):
                print(" ", end="")#tampilkan spasi
        print("|")
        for i in range(garis + 13):
            print(type, end="")#tampilkan garis bawah
        print("")

